test/jasmine/karma.ciconf.js

```diff
@@ -7,6 +7,10 @@ function func(config) {
     func.defaultConfig.logLevel = config.LOG_INFO;
 
     // Continuous Integration mode
+
+    // exclude the WebGL interaction test on circle runs
```

Comment:
why do we have to exclude webgl interaction tests on circle runs? Didn't Docker give us 3D tests?